frontend/scripts/makefile/flutter.toml

```diff
@@ -25,14 +26,39 @@ run_task = { name = [
 [tasks.appflowy-linux]
 dependencies = ["appflowy-core-release"]
 run_task = { name = [
+
   "code_generation",
+
   "set-app-version",
+
   "flutter-build",
+
   "copy-to-product",
+
   "create-release-archive",
 ] }
 script_runner = "@shell"
 
+[tasks.appflowy-android]
+dependencies = ["appflowy-core-dev-android"]
```

Comment:
What's the difference between `appflowy-core-dev-ci-android` and  `appflowy-core-dev-android`?

Reply:
When building ``appflowy-core-dev-ci-android`` it would fail because of the ``$DEST`` value, 
``DEST=${CARGO_MAKE_WORKSPACE_WORKING_DIRECTORY}/app_flowy/android/app/src/main/jniLibs``
It kept failing because of the target directory so ended up creating a separate task for the CI


Appflowy-core-dev-android has the ``$DEST`` value but appflowy-core-dev-ci-android does not.